Add the `v1` API conversion tool to the Operator container image (#12061)
strimzi/strimzi-kafka-operator: v1-api-conversion/README.md

* Patch the CRD resources
* Replace the status of the CRD resources

## Running the conversion tool as a Kubernetes Job

> You can also run the conversion tool as a Kubernetes Job.

Reviewer: ```suggestion
You can run the conversion tool as a Kubernetes Job.
```

> This is useful if you cannot run it locally, or if you, for example, do not have a direct connection to the Kubernetes cluster.

Reviewer: ```suggestion
This approach is useful if you cannot run the tool locally or do not have a direct connection to the Kubernetes cluster.
```
